### PR TITLE
Add local storage of user inputs

### PR DESCRIPTION
**Pull request by mzirkof**
### This Request is about Storing user inputs in local storage (Browser)  
⭐ According to the project requirement, We are asked to : 
▶️ Use Javascript to catch and control the visitor input in the form of the section 'Contact Me ' 👍 
▶️ Use a single object to represent a visitor's information 👍  
▶️Save the information inside the browser using the JS API LocalStorage 👍 
▶️ Read from the saved file the information concerning our visitor and then complete the indicated field in the form 👍 

I hope the 🏈 reaches the 🥅 at a percentage of 💯 

One important thing to consider: I worked alone due to an issue with my programming partner which seems not to be available for many days. So after exchanging with student success, they gave me an answer: to work alone. 
So do not expect a commit from a collaborator (unfortunately 😞 )
Here is a screenshot of the mail : 
Capture

Happy Review their reviewer(s) ☕

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -174,3 +174,45 @@ contactForm.addEventListener('submit', (event) => {
...
+  userDatas.userName = contactForm.user_name.value;
+  localSave();
+});
+contactForm.user_mail.addEventListener('change', () => {
+  userDatas.userMail = contactForm.user_mail.value;
+  localSave();
+});
+contactForm.user_msg.addEventListener('change', () => {
+  userDatas.userMsg = contactForm.user_msg.value;
+  localSave();
```
Comment (topeogunleye) [inline scripts/index.js:217]: - [ ] I kindly suggest that you ensure that your event listeners are working and that the user's input is being saved to local storage as the user types it in. please check my video below for more info:

https://www.loom.com/share/7b02ce5275a64f5f8c696013c99bb359

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @mzirkof, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- In future pull requests, consider following the recommended branch naming conventions, such as using branch names that reflect the specific feature or task being implemented. This will enhance code organization and collaboration within the team. For example: `feature/added-local_storage`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/index.js (hunk 1 above)
Comment (mzirkof) [reply scripts/index.js:228]: Ok Done

**Review by rotimiazeez — APPROVED** [review]
Hi @mzirkof 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job working on the requested changes 👏🏾 
- Data well preserved in the localStorage 😉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**